misc/python/materialize/data_ingest/executor.py

```diff
@@ -122,6 +122,10 @@ def __init__(
             f.result()
             print(f"Topic {topic} created")
 
+        # NOTE: this _could_ be refactored, but since we are fairly certain at
+        # this point there will be exactly one topic it should be fine.
+        topic = list(fs.keys())[0]
```

Comment:
This is also one point in https://github.com/MaterializeInc/materialize/pull/21415, I will double-check the behavior with Dennis when he is back (in one week).